src/graphql/admin/root/query/transaction-by-id.ts

```diff
@@ -13,7 +12,6 @@ const TransactionByIdQuery = GT.Field({
     if (id instanceof Error) throw id
 
     const ledgerTx = await Wallets.getTransactionById(id)
-    if (ledgerTx instanceof CouldNotFindTransactionError) return null
```

Comment:
Why is this clause removed?

Reply:
good question, I don't recall. I see that the type was from @domain so probably the reason. I'll revert for now